source/blog/2016-02-29-up-and-running-with-ovirt-3-6.html.md

```diff
@@ -0,0 +1,310 @@
+---
+title: Up and Running with oVirt 3.6
```

Comment:
It's the minimum configuration for a complete system that can be maintained without shutting all your VMs off -- it's a somewhat historical title, I've been using the same headline for this series since ovirt 3.0, although, early on, it did describe an all in one install. I'm fine w/ you changing the title if you like.